examples/src/components/utils/Header.jsx

```diff
@@ -125,25 +169,97 @@ class Header extends React.Component {
-                    .gd-header-inner :global(.button-logout) {
+                    .gd-header-inner :global(.button-header) {
                         height: 28px;
-                        margin: 0;
+                        margin: 0 0 0 20px;
+                        padding: 0 10px;
+                        border: none;
+                        color: rgba(0, 0, 0, 0.7);
+                        font-size: 12px;
+                        line-height: 28px;
+                    }
+
+                    .gd-header-inner :global(.button-header-border) {
                         padding: 0 22px;
                         border: 1px solid rgba(0, 0, 0, 0.5);
                         border-radius: 100px;
-                        color: black;
-                        font-size: 12px;
                         line-height: 26px;
                     }
 
-                    .gd-header-inner :global(.button-login:hover),
-                    .gd-header-inner :global(.button-login:focus),
-                    .gd-header-inner :global(.button-login:active),
-                    .gd-header-inner :global(.button-logout:hover),
-                    .gd-header-inner :global(.button-logout:focus),
-                    .gd-header-inner :global(.button-logout:active) {
-                        border-color: #000;
+                    .gd-header-inner :global(.button-header:hover),
+                    .gd-header-inner :global(.button-header:focus),
+                    .gd-header-inner :global(.button-header:active) {
+                        color: rgba(0, 0, 0, 0.9);
+                        border-color: rgba(0, 0, 0, 0.9);
+                    }
+
+                    .page :global(.backendInfo) {
+                        position: relative;
+                        display: inline-block;
+                        max-width: 1400px;
+                        margin: 40px 20px 20px;
+                        padding: 15px 60px 15px 30px;
+                        border-radius: 50px;
+                        color: #94a1ad;
+                        font-size: 13px;
+                        text-align: center;
+                        background: #f0f0f0;
+                    }
+
+                    .page :global(.backendInfoItem) {
+                        display: inline-block;
+                        margin-left: 20px;
+                        padding-left: 20px;
+                        border-left: 1px solid #94a1ad;
+                    }
+
+                    .page :global(.backendInfoItem:first-child) {
+                        border-left: none;
+                        margin: 0;
+                        padding: 0;
+                    }
+
+                    .page :global(.backendInfoValue) {
+                        display: inline-block;
+                        margin-left: 4px;
+                        color: #333;
+                    }
+
+                    .page :global(.backendInfoClose) {
```

Comment:
.backendInfoClose doesn't need to be :global

Reply:
Really? ![backendinfo](https://user-images.githubusercontent.com/8083514/41462452-2c9ecc02-7093-11e8-8468-e3ab2e94f216.png)